src/main/java/com/prgrms/needit/domain/user/favorite/controller/FavoriteCenterController.java

```diff
@@ -0,0 +1,49 @@
+package com.prgrms.needit.domain.user.favorite.controller;
+
+import com.prgrms.needit.common.response.ApiResponse;
+import com.prgrms.needit.domain.user.center.dto.CenterListResponse;
+import com.prgrms.needit.domain.user.favorite.service.FavoriteCenterService;
+import java.util.List;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class FavoriteCenterController {
+
+	private final FavoriteCenterService favCenterService;
+
+	public FavoriteCenterController(
+		FavoriteCenterService favCenterService
+	) {
+		this.favCenterService = favCenterService;
+	}
+
+	@GetMapping("/favorites")
+	public ResponseEntity<ApiResponse<List<CenterListResponse>>> getFavCenters() {
+		return ResponseEntity.ok(
+			ApiResponse.of(favCenterService.getFavCenters())
+		);
+	}
+
+	@PostMapping("centers/{centerId}/favorites")
+	public ResponseEntity<ApiResponse<Long>> addFavoriteCenter(
+		@PathVariable Long id
+	) {
+		return ResponseEntity.ok(
+			ApiResponse.of(favCenterService.addFavoriteCenter(id))
+		);
+	}
```

Comment:
센터에 종속되어있는 것 같아보이니 저는 아래와 같이 해도 될 것 같은데 하정님의 생각은???

```suggestion
	@PostMapping("/favorites")
	public ResponseEntity<ApiResponse<Long>> addFavoriteCenter(
		@PathVariable Long id
	) {
		return ResponseEntity.ok(
			ApiResponse.of(favCenterService.addFavoriteCenter(id))
		);
	}
```

Reply:
아하 저는 "센터 상세조회 페이지" 에서 addFavoriteCenter가 가능한 거라서 일부러 넣었는데 없애도 된다면 없애겠습니다!

Reply:
앗 근데 생각해보니 centerId를 path variable 로 받고있어서...!!

Reply:
오홍... 일단 이렇게 가고 더 나은 api를 생각해봅시당

Reply:
팀미팅 후 favorites/{centerId}로 바꿨습니다!